packages/api-kit/src/SafeApiKit.ts

```diff
@@ -282,11 +284,11 @@ class SafeApiKit {
     if (label) {
       url.searchParams.set('label', label)
     }
-    if (limit) {
-      url.searchParams.set('limit', limit)
+    if (limit != null) {
```

Comment:
Why this? what happen if it's undefined or false?

Reply:
This is to allow the value being `0`. With the condition we had before, it would not set the search param. 
Now it would not be set for undefined either. For false it would be set. But the prop's type is number, so this should not happen.